circle.yml

```diff
@@ -0,0 +1,17 @@
+machine:
+  node:
+    version: v0.12.16
+
+dependencies:
+  pre:
+    - nvm install v0.10.47
+    - nvm install v0.12.16
+    - nvm install v4.6
```

Comment:
I know - and I wanted to keep specific 0.10 + 0.12 versions specific, because they can't bump minor versions for any breaking-changes (one of those was introduction of `npm@2` in one of ~0.10.45 version)

And this code is not going to remain here for a long time, I guess.